core/src/main/java/com/auth0/authentication/AuthenticationAPIClient.java

```diff
@@ -447,7 +447,7 @@ public ChangePasswordRequest changePassword(String email) {
                 .addParameters(parameters);
     }
 
-    protected ParameterizableRequest<Token> loginWithResourceOwner() {
+    public ParameterizableRequest<Token> loginWithResourceOwner() {
```

Comment:
Ok no harm to keep it but don't constraint the design of the library to Lock.Android yet